plugins/active_users/tests.js

```diff
@@ -0,0 +1,85 @@
+var request = require('supertest');
+var should = require('should');
+var testUtils = require("../../test/testUtils");
+request = request.agent(testUtils.url);
+
+var APP_KEY = "";
+var API_KEY_ADMIN = "";
+var APP_ID = "";
+
+describe('Testing Active Users', function() {
+    describe('Reset app', function() {
+        it('should reset data', function(done) {
+            API_KEY_ADMIN = testUtils.get("API_KEY_ADMIN");
```

Comment:
You don;t need to do it for each test. Just do it once on the first one.
(Reassigining API_KEY_ADMIN + APP_ID values)